app/src/main/java/org/oppia/android/app/application/BUILD.bazel

```diff
@@ -120,7 +120,6 @@ android_library(
         "//domain/src/main/java/org/oppia/android/domain/oppialogger/exceptions:logger_module",
         "//domain/src/main/java/org/oppia/android/domain/oppialogger/logscheduler:metric_log_scheduler_module",
         "//domain/src/main/java/org/oppia/android/domain/oppialogger/loguploader:worker_module",
-        "//domain/src/main/java/org/oppia/android/domain/survey:question_constants_module",
```

Comment:
hhggas

Reply:
This was an erroneous comment from @kkmurerwa